src/sass/partials/components/_forms.scss

```diff
@@ -24,6 +24,34 @@
     }
   }
 
+  &--add {
+    @include button(setting-get('primary accent color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary color'), setting-get('primary color'));
+
+    background-image: url('/images/icons/add.svg');
+    background-position: setting-get('rhythm') 50%;
+    background-repeat: no-repeat;
+    background-size: 1em;
+    margin-top: setting-get('rhythm') * 2;
+    padding-left: setting-get('rhythm') * 2.5;
+  }
+
+  &--delete {
+    @include link(setting-get('primary color'), setting-get('primary color'), setting-get('secondary link color'), setting-get('secondary link color'));
+
+    background-color: transparent;
+    border: {
+      left: 0 none;
```

Comment:
If the width of all of these is `0` then they don't need their style set to `none`